### PR TITLE
Clean `sendop` `recv` operator.

### PR DESCRIPTION
**Pull request by gongweibao**
Fix https://github.com/PaddlePaddle/Paddle/issues/11304

### CODE REVIEW

File: paddle/fluid/operators/recv_vars_op.cc [hunk 1]
```diff
@@ -26,11 +26,11 @@ limitations under the License. */
 namespace paddle {
 namespace operators {
 
-class RecvOp : public framework::OperatorBase {
+class RecvVarsOp : public framework::OperatorBase {
  public:
-  RecvOp(const std::string& type, const framework::VariableNameMap& inputs,
-         const framework::VariableNameMap& outputs,
-         const framework::AttributeMap& attrs)
+  RecvVarsOp(const std::string& type, const framework::VariableNameMap& inputs,
```
Comment (typhoonzero) [inline paddle/fluid/operators/recv_vars_op.cc]: We may call them send, recv op is OK now.

Comment (gongweibao) [reply paddle/fluid/operators/recv_vars_op.cc]: Done. Thanks!

File: paddle/fluid/operators/send_op.cc [hunk 2]
```diff
@@ -36,12 +35,9 @@ class SendOp : public framework::OperatorBase {
...
                const platform::Place& place) const override {
     auto ins = Inputs("X");
-    auto outs = Outputs("Out");
-    std::vector<std::string> epmap = Attr<std::vector<std::string>>("epmap");
-    std::vector<std::string> endpoints =
-        Attr<std::vector<std::string>>("endpoints");
 
-    bool sync_mode = Attr<bool>("sync_mode");
+    std::vector<std::string> epmap = Attr<std::vector<std::string>>("epmap");
+    int sync_send = Attr<int>("sync_send");
```
Comment (typhoonzero) [inline paddle/fluid/operators/send_op.cc]: Keep the name "sync_mode" is better because it's the same as in the transpiler.

Comment (gongweibao) [reply paddle/fluid/operators/send_op.cc]: Done. Thanks!

**Pull request merged**